node/service/src/lib.rs

```diff
@@ -1219,6 +1221,14 @@ where
 		} else {
 			task_manager.spawn_handle().spawn_blocking("beefy-gadget", None, gadget);
 		}
+
+		if is_offchain_indexing_enabled {
+			task_manager.spawn_handle().spawn_blocking(
+				"mmr-gadget",
+				None,
+				MmrGadget::start(client.clone(), backend.clone(), b"mmr".to_vec()),
```

Comment:
can we move this `b"mmr"` as const defined in `primitives` and use that for both the pallet and the gadget.

Reply:
Good idea. Done.